# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../embutils/build.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Remove some unusal build system stuff, so that we can build it normally,
linked against any libc ...

  - Rene Rebe <rene@exactcode.de>

--- embutils-0.17/Makefile	2005-01-31 19:06:38.000000000 +0100
+++ embutils-0.17-patched/Makefile	2005-02-02 12:54:27.383231704 +0100
@@ -1,9 +1,7 @@
 INSTALL=install
-prefix=/opt/diet
+prefix=/usr
 # Set the following to install to a different root
 #DESTDIR=/tmp/fefix
-# Set the following to use the diet libc
-DIET=diet -Os
 
 FLAGS=
 
@@ -15,30 +13,26 @@
 truncate strings test date mount printenv umount pivot_root insmod rmmod \
 lsmod
 
-ARCH:=$(shell uname -m | sed 's/i[4-9]86/i386/')
-
-OBJDIR:=bin-$(ARCH)
+OBJDIR:=bin
 TARGETS=$(patsubst %,$(OBJDIR)/%,$(PRGS))
 
 all: $(OBJDIR) $(TARGETS)
 
 CFLAGS=-pipe -Os -fomit-frame-pointer
 
-CROSS=
-
 #CFLAGS=-pipe -g
 
 
 $(OBJDIR)/%.o: %.c
-	$(DIET) $(CROSS)gcc $(CFLAGS) $(FLAGS) -c $< -o $@
-	$(CROSS)strip -x -R .comment -R .note $@
+	$(CC) $(CFLAGS) $(FLAGS) -c $< -o $@
+	$(STRIP) -x -R .comment -R .note $@
 
 $(OBJDIR)/%.o: %.S
-	$(DIET) $(CROSS)gcc -pipe -c $< -o $@
+	$(CC) -pipe -c $< -o $@
 
 $(OBJDIR)/%: $(OBJDIR)/%.o
-	$(DIET) $(CROSS)gcc -o $@ $^
-	$(CROSS)strip -R .note -R .comment $@
+	$(CC) -o $@ $^
+	$(STRIP) -R .note -R .comment $@
 
 $(OBJDIR)/uniq: $(OBJDIR)/uniq.o $(OBJDIR)/fmt_ulongpadright.o $(OBJDIR)/fmt_ulong.o
 
